### DISCUSSION TITLE
Can I still use Office-UI-Fabric-Core

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#24124] [Q&A] [answered]
Hi,

I would like to style some custom HTML with FluentUI styles.  Should I use the office-ui-fabric-core package or do I need to use another package for purely accessing fluentui CSS styles.

Thanks.

**Comment by micahgodbolt** [comment, marked as answer]
The core package is not currently maintained, but it is still available for those that still need it.
